modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -332,6 +356,10 @@ export class Tileset3D {
     return new URLSearchParams(this._queryParams).toString();
   }
 
+  get memoryAdjustedScreenSpaceError(): number {
```

Comment:
Nit: I am not a fan of getters. I know Cesium uses them extensively, but I think they add more clutter than they are worth for enforcing a semi-read-ony value.